express-notes-api/index.js

```diff
@@ -0,0 +1,106 @@
+const express = require('express');
+const app = express();
+const notebookData = require('./data.json');
+const fs = require('fs');
+
+const notebook = notebookData.notes;
+const noteId = notebookData.nextId;
```

Comment:
This is causing your server to erase the previously saved note every time. Try just using `notebook.nextId` directly in your POST route.